### DISCUSSION TITLE
Is 2.0.0-beta4.22272.1 obsolete?

### DISCUSSION THREAD

**Discussion by rjdunnill** [dotnet/command-line-api#2190] [Q&A]
I installed 2.0.0-beta4.22272.1 via NuGet, but looking at the code in the repositories here, the architecture of System.CommandLine appears to have changed radically.

Does this mean the current released version's architecture is to be retired?

**Comment by jonsequitur** [comment]
Yes, the current changes will be released in a new beta in the coming weeks. The changes are intended to bring the API into line with the .NET framework design guidelines and reduce the complexity of the API, but there are few if any reductions in capabilities as a result.
